SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -135,5 +157,49 @@ public override bool InstanceEnabled
                 }
             }
         }
+
+        private void ProcessPendingErrors()
+        {
+            foreach (var file in ErrorLogHelper.GetErrorLogFiles())
+            {
+                AppCenterLog.Debug(LogTag, $"Process pending error file {file.Name}");
+                var log = ErrorLogHelper.ReadErrorLogFile(file);
+                if (log == null)
+                {
+                    AppCenterLog.Error(LogTag, $"Error parsing error log. Deleting invalid file: {file.Name}");
+                    try
+                    {
+                        file.Delete();
```

Comment:
Doesn't ErrorLogHelper have a method to delete errors, which will also take care of exceptions?

Reply:
This is handling deserialization issue. We don't have an error ID so it removes the file from here.